discord-scripts/fix-twitter-embed.ts

```diff
@@ -62,27 +67,42 @@ async function workingTwitterEmbeds(
 //
 // See https://github.com/FixTweet/FxTwitter for more.
 export default function fixTwitterEmbeds(discordClient: Client, robot: Robot) {
+  // Process only messages that match the Twitter URL pattern
   discordClient.on("messageCreate", (message) => {
-    robot.logger.info(
+    robot.logger.debug(
       `fixTwitterEmbeds: processing new message ${message.content}`,
     )
-
-    workingTwitterEmbeds(message, robot.logger).catch((err) => {
-      robot.logger.error(
-        `fixTwitterEmbeds: failed to process new message ${message.content}: ${err}`,
+    if (message.content?.match(twitterUrlRegExp)) {
+      robot.logger.info(
+        `fixTwitterEmbeds: processing new message ${message.content}`,
       )
-    })
+      workingTwitterEmbeds(message, robot.logger).catch((err) => {
+        robot.logger.error(
+          `fixTwitterEmbeds: failed to process new message ${message.content}: ${err}`,
+        )
+      })
+    }
   })
 
   discordClient.on("messageUpdate", (oldMessage, newMessage) => {
-    robot.logger.info(
-      `fixTwitterEmbeds: processing updated message ${newMessage.content}, ${oldMessage.content}`,
+    robot.logger.debug(
+      `fixTwitterEmbeds: processing updated message ${newMessage.content}`,
     )
+    if (
+      newMessage.content?.match(twitterUrlRegExp) ||
+      oldMessage?.content?.match(twitterUrlRegExp)
+    ) {
+      robot.logger.info(
+        `fixTwitterEmbeds: processing updated message ${newMessage.content}`,
```

Comment:
Looks like we now log this twice? And same note as above re: leaving message content out of the info level.